### ISSUE TITLE
Reduce multistats API calls

### ISSUE THREAD

**Issue by nikola-rados** [pacificclimate/p2a-rule-engine#4]
Currently the rule engine iterates over a list of `models` and makes a `multistats` call for each iteration.  By making the call without specifying a `model` it _should_ return data for all the `models`.  This would massively reduce the number of calls we make for non-historical data.  Work will have to be done to handle processing the new output, some details can be found in [this comment](https://github.com/pacificclimate/p2a-rule-engine/issues/3#issuecomment-584424318).

This is a sub-issue of issue #3.

**Comment by nikola-rados** [comment]
I wrote a quick script that tested the speed of the `multistats` call.  I used a hard coded set of `query_args` and the `p2a_rcp85` ensemble.  These tests were conducted on my local machine and on leopard.  

Using the current query strategy of looping over each model and hitting the backend the speed was:
```
(venv) p2a-rule-engine$ time python scripts/test_api_calls.py --run-type multi

real	0m16.500s
user	0m10.683s
sys	0m1.566s
```

Using the proposed strategy of a single call for all models:
```
(venv) p2a-rule-engine$ time python scripts/test_api_calls.py --run-type single

real	0m16.083s
user	0m10.748s
sys	0m1.615s
```

There is hardly a difference.  Furthermore, the results on leopard were only 1-2 seconds faster.  As things currently stand I don't believe this is an avenue worth pursuing.
